app/assets/javascripts/search.js

```diff
@@ -1,18 +1,19 @@
 $(document).ready(function() {
 
-  var searchUI = $(".m-search-ui");
-  var advOptsFieldset = $("fieldset.adv");
-  var advOptsControlsFieldset = $("fieldset.controls");
-  var searchTerms = $(".m-search-ui .search-terms");
-  var searchButton = $("#search-button");
-  var searchMagGlass = $(".m-search-ui .input-group-addon.magnifier");
-  var advOptsButton = $("#adv-options");
-  var advOptsToggler = $("a.adv-options");
-  var advOptsCloser = $(".adv-controls .closer");
-  var allSearchButtons = $(".m-search-ui button.search");
-  var searchForm = $("form.search");
-  var countEl = $(".results-count-preview .count");
-  var advOptsResetter = $("a.resetter");
+  var searchUI = $(".m-search-ui"),
+  advOptsFieldset = $("fieldset.adv"),
+  advOptsControlsFieldset = $("fieldset.controls"),
+  searchTerms = $(".m-search-ui .search-terms"),
+  searchButton = $("#search-button"),
+  searchMagGlass = $(".m-search-ui .input-group-addon.magnifier"),
+  advOptsButton = $("#adv-options"),
+  advOptsToggler = $("a.adv-options"),
+  advOptsCloser = $(".adv-controls .closer"),
+  allSearchButtons = $(".m-search-ui button.search"),
+  searchForm = $("form.search"),
+  countEl = $(".results-count-preview .count"),
+  advOptsResetter = $("a.resetter"),
+  formHasFocus = false;
```

Comment:
Correct it's just a syntax thing. They are anonymous

On Tuesday, September 6, 2016, Leonard Bogdonoff <notifications@github.com>
wrote:

> In app/assets/javascripts/search.js
> <https://github.com/18F/C2/pull/1497#discussion_r77745251>:
>
> > -  var countEl = $(".results-count-preview .count");
> > -  var advOptsResetter = $("a.resetter");
> > +  var searchUI = $(".m-search-ui"),
> > +  advOptsFieldset = $("fieldset.adv"),
> > +  advOptsControlsFieldset = $("fieldset.controls"),
> > +  searchTerms = $(".m-search-ui .search-terms"),
> > +  searchButton = $("#search-button"),
> > +  searchMagGlass = $(".m-search-ui .input-group-addon.magnifier"),
> > +  advOptsButton = $("#adv-options"),
> > +  advOptsToggler = $("a.adv-options"),
> > +  advOptsCloser = $(".adv-controls .closer"),
> > +  allSearchButtons = $(".m-search-ui button.search"),
> > +  searchForm = $("form.search"),
> > +  countEl = $(".results-count-preview .count"),
> > +  advOptsResetter = $("a.resetter"),
> > +  formHasFocus = false;
>
> @dogweather <https://github.com/dogweather> There is a comma at the end
> of each assignment, so the first var is being applied to everything that
> follows.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/18F/C2/pull/1497/files/b370bf6d565c5333cee53d090939e731615cb4e2#r77745251>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFDQeBFBOg3BGHpwFDfQaVt6RHEkNtcQks5qngs-gaJpZM4J2L_V>
> .
>


-- 
Nick Bristow
18F Innovation Specialist
Nick.Bristow@gsa.gov